tools/packages.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-    <package id="Cake" version="0.26.1" />
+    <package id="Cake" version="0.30.0" />
```

Comment:
Despite what I said above in my other comment, having the build of this addin use 0.30.0 is perfectly fine 👍

Reply:
OK, I figured newer was better but this makes sense. Updated the PR.